sdk/metric/controller/push/config.go

```diff
@@ -33,6 +34,10 @@ type Config struct {
 	// integrate, and export) can last before it is canceled. Defaults to
 	// the controller push period.
 	Timeout time.Duration
+
+	// MetricsLabelsEnricher is a function that enriches metrics labels based
+	// on kvs stored in context when metrics are recorded.
+	MetricsLabelsEnricher metric.MetricsLabelsEnricher
```

Comment:
(Repeat: https://github.com/open-telemetry/opentelemetry-specification/issues/1207 says we can and should consolidate push and pull controllers, to avoid this repetition.)